### PR TITLE
pasta: missing setuptools dep

### PR DESCRIPTION
**Pull request by snehring**
pasta needs setuptools. I guess the python I was using last week to build this had that already, caught it today in a fresh python.

### CODE REVIEW

**Review by tldahlgren — DISMISSED** [review]
LGTM.  Will give @adamjstewart a chance to comment before merging.

File: var/spack/repos/builtin/packages/pasta/package.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class Pasta(Package):
     depends_on("python@3:", when="@1.9.0:")
     depends_on("python@:2", when="@:1.8.3")
     depends_on("py-dendropy")
+    depends_on("py-setuptools")
     depends_on("java")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pasta/package.py]: Guessing this should look something like:
```suggestion
    depends_on("python@3:", when="@1.9:", type=("build", "run"))
    depends_on("python@:2", when="@:1.8", type=("build", "run"))
    depends_on("py-dendropy@4:", type=("build", "run"))
    depends_on("py-setuptools", type="build")
    depends_on("java", type=("build", "run"))
```

File: var/spack/repos/builtin/packages/pasta/package.py [hunk 2]
```diff
@@ -38,7 +39,5 @@ def install(self, spec, prefix):
         install_tree(".", prefix)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pasta/package.py]: Can you try removing the `install` method and using `PythonPackage`? Would like to change this for the same reasons as busco, we don't want to install all these READMEs, docs, tests, etc.

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/pasta/package.py (hunk 1 above)
Comment (snehring) [reply var/spack/repos/builtin/packages/pasta/package.py:23]: I'll try, or at the very least make the same changes we had with busco.

File: var/spack/repos/builtin/packages/pasta/package.py (hunk 2 above)
Comment (snehring) [reply var/spack/repos/builtin/packages/pasta/package.py]: I'll try, or at the very least make the same changes we had with busco.

**Review by adamjstewart — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**